src/FuncHW/Program.cs

```diff
@@ -27,14 +27,21 @@ static void Main()
 
             //Func*** funcPersonIsActive = ***;
             //Func*** funcPersonPhoneNumberStartsWith7 = ***;
-            //Func*** funcPersonIsChild = ***;
+
+            Func<Person, bool> funcPersonIsChild = (Person person) => DateTime.Now.Year - person.BirthDate.Year > 17;
+
             //Func*** funcPersonHasShortName = ***;
 
-            //Func*** funcCatIsDomestic = ***;
+            Func<Cat, bool> funcCatIsDomestic = (Cat cats) => cats.IsDomestic;
+
             //Func*** funcCatColorIsDark = ***;
             //Func*** funcCatNameConteinsU = ***;
             //Func*** funcCatIsDomesticAndWhite = ***;
 
+            int res1 = ListExtensions.CountElements(persons, funcPersonIsChild);
```

Comment:
Name variables according to value it contains. e.x. `countPersonIsChild` or `countChildrenInPersons` etc.